doc/extensions/acrbuildext/azext_acrbuildext/__init__.py

```diff
@@ -42,4 +55,19 @@ def load_arguments(self, _):
             c.argument('no_logs', options_list=['--no-logs'], action='store_true', help='Do not show logs after successfully queuing the build.') 
         return self.argument_context
 
+        with self.argument_context('acr build-task') as c:
+            c.argument('registry_name', options_list=['--registry', '-r'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
+            c.argument('build_task_name', options_list=['--name', '-n'], help='The name of the build task', completer=get_resource_name_completion_list(BUILD_TASK_RESOURCE_TYPE))
+            c.argument('image_name', options_list=['--image', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
+            c.argument('context', options_list=['--context'], help="The URL to a git repository.")
+            c.argument('source_branch', options_list=['--branch'], help="The source control branch name")
+            c.argument('docker_file_path', options_list=['--file', '-f'], help="The relative path of the the docker file to the source code root folder.")
+            c.argument('git_access_token', help="The git access token for configuring webhook.")
+            c.argument('os_type', help="The platform OS that has to be used for build task.")
```

Comment:
NIT: I think we can add a `options_list=['--os']` here to simplify the parameter.